login_server/src/cpp/JSONInterface/JsonGetUsers.cpp

```diff
@@ -93,10 +93,10 @@ Poco::JSON::Object* JsonGetUsers::handle(Poco::Dynamic::Var params)
 	result->set("state", "success");
 
 	//Poco::JSON::Object jsonResultObject;
-	Poco::JSON::Array  jsonUsersArray;
+	Poco::JSON::Array::Ptr jsonUsersArray = new Poco::JSON::Array;
```

Comment:
Why change the type to Pointer? How is this related to the SQL problem?

Reply:
It is not related to SQL Problems, but I need as a type Pointer for testing.